j5/boards/sr/v4/power_board.py

```diff
@@ -0,0 +1,126 @@
+"""Classes for the SR v4 Power Board."""
+
+from enum import Enum
+from typing import TYPE_CHECKING, List, Union, cast, Mapping
+
+from j5.backends import Backend, Environment
+from j5.boards import Board
+from j5.components import LED, BatterySensor, Button, Piezo, PowerOutput
+
+if TYPE_CHECKING:
+    from j5.components import (  # noqa: F401
+        Component,
+        ButtonInterface,
+        PowerOutputInterface,
+        PiezoInterface,
+        BatterySensorInterface,
+        LEDInterface,
+    )
+    from typing import Type  # noqa: F401
+
+
+class PowerOutputPosition(Enum):
+    """A mapping of name to number of the PowerBoard outputs."""
```

Comment:
I think its worth documenting here that the numbers are the same as are used in the wire protocol, to prevent anyone deciding to renumber the outputs in the future.